linux_sql/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
do not include .idea files (these are your IDE config files. Other people do not need them)